modules/local_storage/commerce_stock_local.install

```diff
@@ -293,3 +301,17 @@ function commerce_stock_local_install() {
     ])
     ->execute();
 }
+
```

Comment:
That's cool but do we need updates for pre beta versions, not complaining just wondering ?